packages/data-access-plugin/src/manager/types/data-type.ts

```diff
@@ -1,26 +1,38 @@
 import { baseModel } from './misc';
 
 export default `
+    type DataTypeConfig {
+        fields: DataTypeFields
+        version: Int
+    }
+
+    input DataTypeConfigInput {
+        fields: DataTypeFields
```

Comment:
Fields should be required
```suggestion
        fields: DataTypeFields!
```